### ISSUE TITLE
Facebook and FacebookWeb Provider Crashing

### ISSUE THREAD

**Issue by rcandil** [calebd/SimpleAuth#122]
Hi, I'm new to SimpleAuth. I started with Facebook provider, but no joy. It crashes at line:

```
dictionary[@"urls"] = @{
    @"Facebook": remoteAccount[@"link"]
```

at file: SimpleAuthFacebookProvider.m 

with error: 
2016-01-10 19:25:26.220 loginSimpleAuth[352:80534] **\* Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '**\* -[__NSPlaceholderDictionary initWithObjects:forKeys:count:]: attempt to insert nil object from objects[0]'

When inspecting the "remoteAccount" dictionary, I just find two key/values: one for "id" and the other for "name", there is nothing else in it.

I also tried with FacebookWeb provider as well, when it returns from the login page from Facebook, my App crashes with the same error message.

Kindly help me

**Comment by adamjmcgrath** [comment]
There is a fix for this in #109 

You can use my fork in your Podfile:

`pod 'SimpleAuth/Facebook', :git => 'https://github.com/adamjmcgrath/SimpleAuth-1.git', :branch => 'facebook-2.4-breaking-changes'`

**Comment by rcandil** [comment]
Thanks, it is now working just fine
